CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Changed
+
+- Use resource.Quantity.AsApproximateFloat64() instead of AsInt64(), in order to avoid conversion issue when multiply cpu, e.g. 3880m
+
```

Comment:
I'm not sure I understand the problematic case.
* It fails at multiplying `3880*0.9`? That's a round number - 3492
* It fails at multiplying `3.880x0.9`? Then, why it didn't fail when multiplying `4x0.9`?